Dockerfiles/Python/Dockerfile

```diff
@@ -24,6 +24,12 @@ RUN apt-get update && apt-get upgrade -y && \
         libstdc++6 zlib1g locales sudo ncdu man-db strace gnupg2 \
         build-essential python3-dev
 
+# Install CLI
+ARG TARGETARCH
+COPY ./bin/velocitas-cli-$TARGETARCH /usr/bin/velocitas
+RUN chmod +x /usr/bin/velocitas
```

Comment:
can't we do this before commiting? or is it not persistent?

Reply:
Changed file permissions and removed this